### ISSUE TITLE
Linter flagging required parameters

### ISSUE THREAD

**Issue by patricknelson** [micantoine/svelte-preprocess-cssmodules#94]
Fairly minor, but it appears TypeScript is flagging `cssModules()` arguments as being required when in fact they're actually optional:

> TS2554: Expected 1 arguments, but got 0.

![image](https://user-images.githubusercontent.com/4269377/235036685-89212c08-bcb0-4d65-962e-372df858df9b.png)

**Comment by micantoine** [comment]
Yes, there is a typing mistake, the parameter is still not optional

Instead of 

```js
cssModules(options: Partial<PluginOptions>): PreprocessorGroup
```

it should be

```js
cssModules(options: Partial<PluginOptions> = {}): PreprocessorGroup
```

for the time being, you can pass an empty object `cssModules({})`

**Comment by micantoine** [comment]
fixed on `v2.2.5`
